solutions/src/main/java/ru/sberbank/school/task08/SerializableManager.java

```diff
@@ -17,17 +19,42 @@ public SerializableManager(@NonNull String filesDirectoryPath) {
```

Comment:
&#39;CTOR_DEF&#39; should be separated from previous statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-985261138>*